crud-operations/src/main/java/org/example/crudoperations/controllers/ProductController.java

```diff
@@ -0,0 +1,44 @@
+package org.example.crudoperations.controllers;
+
+import org.example.crudoperations.entity.Product;
+import org.example.crudoperations.service.ProductService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+@RestController
+@CrossOrigin(origins ="${client.url}")
+public class ProductController {
+    @Autowired
+    private ProductService productService;
+
+    @PostMapping("/addProduct")
+    public Product addProduct(@RequestBody Product product){
```

Comment:
same here. You can use [mappers](https://mapstruct.org/) for converting custom pojo to entity in service